src/Acuminator/Acuminator.Analyzers/StaticAnalysis/SavingChanges/SaveOperationHelper.cs

```diff
@@ -61,6 +62,28 @@ internal static class SaveOperationHelper
 			return SaveOperationKind.None;
 		}
 
+		public static PXDatabaseKind GetPXDatabaseSaveOperationKind(IMethodSymbol symbol, PXContext pxContext)
```

Comment:
This helper and the enums it uses should be moved to utilities project. It's generic shareable functionality